src/components/toast/global_toast_list.tsx

```diff
@@ -283,6 +305,42 @@ export const EuiGlobalToastList: FunctionComponent<EuiGlobalToastListProps> = ({
     );
   });
 
+  if (showClearAllButtonAt && toasts.length >= showClearAllButtonAt) {
+    const dismissAllToastImmediately = () => {
```

Comment:
[very optional, incredibly minor grammar nit]
```suggestion
    const dismissAllToastsImmediately = () => {
```
Also another optional suggestion b/c naming things is hard - what do you think about removing the `Immediately` adverb for brevity?
```suggestion
    const dismissAllToasts = () => {
```